test/663-combo-outdoor-landuse-pois.py

```diff
@@ -0,0 +1,392 @@
+#http://www.openstreetmap.org/way/31198945
+# Waterworld in Concord
+assert_has_feature(
+    13, 1318, 3160, 'landuse',
+    { 'kind': 'water_park', 'sort_key': 102 })
+
+assert_has_feature(
+    13, 1318, 3160, 'pois',
+    { 'kind': 'water_park', 'min_zoom': 13 })
+
+#http://www.openstreetmap.org/node/2753215890
+# Antioch WaterPark
+assert_has_feature(
+    16, 10600, 25287, 'pois',
+    { 'kind': 'water_park', 'min_zoom': 15 })
+
+
+
+#http://www.openstreetmap.org/way/381817396
+# Sandos Finisterra Los Cabos, extra large resort
+assert_has_feature(
+    14, 3189, 7122, 'pois',
+    { 'kind': 'beach_resort', 'min_zoom': 14 })
+
+#http://www.openstreetmap.org/node/2407351205
+# Hilton Hawaiian Village
+# really this should be merged with its large AOI!
+assert_has_feature(
+    16, 4034, 28801, 'pois',
+    { 'kind': 'beach_resort', 'min_zoom': 16 })
+
+#http://www.openstreetmap.org/node/3575812477
+# Silver Gull Beach Club
+assert_has_feature(
+    16, 19314, 24677, 'pois',
+    { 'kind': 'beach_resort', 'min_zoom': 16 })
+  
+#https://www.openstreetmap.org/node/1500943741
+# Needs to appear **before** tourism=hotel
+# Best Western Plus Blue Sea Lodge
+assert_has_feature(
+    16, 11422, 26443, 'pois',
+    { 'kind': 'beach_resort', 'min_zoom': 16 })
+
+
+#https://www.openstreetmap.org/way/257716817
+# Needs to appear **after** natural=beach.
+# Unnamed beach in Maskenthine Lake area
+assert_has_feature(
+    16, 15066, 24333, 'pois',
+    { 'kind': 'beach', 'id': 257716817 })
+
+#https://www.openstreetmap.org/way/257716817
+# Needs to appear **after** natural=beach.
+# Unnamed beach in Maskenthine Lake area
+assert_has_feature(
+    16, 15066, 24333, 'landuse',
+    { 'kind': 'beach', 'id': 257716817 })
+
+
+
+#https://www.openstreetmap.org/node/3655879348
+# Camp Ahmek
+assert_has_feature(
+    15, 9219, 11714, 'pois',
+    { 'kind': 'summer_camp', 'min_zoom': 15 })
+    
+#https://www.openstreetmap.org/node/4050178586
+# missing
+# Camp Goodtimes
+# assert_has_feature(
+#     15, 5225, 11211, 'pois',
+#     { 'kind': 'summer_camp', 'min_zoom': 15 })    
+
+
+
+#https://www.openstreetmap.org/node/3838356961
```

Comment:
I'm seeing the Battle of Blackburn's Ford battlefield POI OK:

```json
    {
      "geometry": {
        "type": "Point",
        "coordinates": [
          -77.44954723,
          38.80331938
        ]
      },
      "type": "Feature",
      "properties": {
        "kind": "battlefield",
        "min_zoom": 17,
        "id": 3838356961,
        "name": "Battle of Blackburn's Ford (1861)"
      }
    }
```

I'm missing the Kernstown POI from my local DB, so couldn't test that one.

The Antietam battlefield is getting a `min_zoom` of 10.4417 in my local DB, which means it only kicks in at zoom 11 (in tile 11/581/779):

```json
    {
      "geometry": {
        "type": "Point",
        "coordinates": [
          -77.7362,
          39.4678
        ]
      },
      "type": "Feature",
      "properties": {
        "id": 231393152,
        "kind": "battlefield",
        "min_zoom": 10.4417,
        "name": "Antietam National Battlefield",
        "area": 12969200
      }
    },
```

The polygon is also present in `landuse` (again, at zoom 11), but as a `national_park`:

```json
      "type": "Feature",
      "properties": {
        "kind": "national_park",
        "source": "openstreetmap.org",
        "sort_key": 18,
        "area": 12969200,
        "mz_is_building": false,
        "min_zoom": 5.44171,
        "id": 231393152
      }
```

The filter for `boundary=national_park` => `kind: national_park` is the highest one in the YAML file, so take priority.